.golangci.yml

```diff
@@ -0,0 +1,69 @@
+linters-settings:
+  govet:
+    check-shadowing: true
+  golint:
+    min-confidence: 0
+  maligned:
+    suggest-new: true
+  misspell:
+    locale: UK
+  gocritic:
+    enabled-tags:
+      - performance
+      - style
+      - experimental
+    disabled-checks:
+      - sloppyReassign
+      - docStub
+      - paramTypeCombine
+
+linters:
+  enable-all: true
+  disable:
+    - dupl
+    - gochecknoglobals
+    - gocyclo
+    - goconst
+    - lll
+    - misspell
+
+issues:
+  exclude-rules:
+    #Ignore variable scope linting
+    - path: "errors/error_handler_test.go"
+      linters:
+        - scopelint
+    - path: "config/config_test.go"
+      linters:
+        - scopelint
+    # Ignore things like "FormId should be FormID"
+    - path: codegen
+      text: "ST1003:"
+      linters:
+        - stylecheck
+    # We like underscores
+    - path: codegen
+      text: "don't use underscores in Go names"
+      linters:
+        - golint
+    # Ignore things like "FormId should be FormID"
+    - path: codegen
+      text: ".*should be.*"
+      linters:
+        - golint
+    # ignore err shadowing in test files
+    - path: _test\.go
+      linters:
+        - govet
+
+run:
+  deadline: 2m
+  build-tags:
+    - codeanalysis
+
+# golangci.com configuration
+# https://github.com/golangci/golangci/wiki/Configuration
+service:
+  golangci-lint-version: 1.19.x # use the fixed version to not introduce new linters unexpectedly
+  prepare:
+    - echo "here I can run custom commands, but no preparation needed for this repo"
```

Comment:
Remove the line above.

Reply:
Removed the entire service section, thanks!